packages/babel-core/src/config/validation/plugins.ts

```diff
@@ -51,10 +51,9 @@ function assertVisitorMap(loc: OptionPath, value: unknown): Visitor {
   return obj as Visitor;
 }
 
-function assertVisitorHandler(
-  key: string,
-  value: unknown,
-): VisitorHandler | void {
+function assertVisitorHandler(key: string, value: unknown) {
+  if (key === "_exploded" || key === "_verified") return;
```

Comment:
We explicitly disallow catch-all `enter` and `exit` in Babel plugins:

https://github.com/babel/babel/blob/2db3f556a4f36f4d261f671984f107a73aa9eab3/packages/babel-core/src/config/validation/plugins.ts#L43-L49

If `key` is `_exploded`, then `value` is a visitor processed by `explode`. In this case, I think we should move the catch-all check before the visitor shape check, so we don't have to handle `_exploded` and `_verified` in `assertVisitorHandler`.